docs/source/conf.py

```diff
@@ -148,24 +147,6 @@
 
 # -- Options for LaTeX output ---------------------------------------------
 
-latex_elements = {
```

Comment:
No need to delete this, we might actually start to add latex formulas and it's good to have something to start with

Reply:
Well let me revert.